src/components/RadioElement.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import RoundNumber from "./RoundNumber";
+
+import { useBreakpoint } from "../helpers/hooks";
+import breakpoint from "../helpers/breakpoint";
+
+import consts from "../consts.js";
+
+import music from "../img/type/1music.png";
+import speech from "../img/type/2speech.png";
+import ads from "../img/type/3ads.png";
+
+const Shadow = styled.div`
+	opacity: 0.5;
+	width: 100%;
+	height: 100%;
+	border-radius: 50%;
+	border: 1px solid black;
+
+	${props =>
+		props.playing && breakpoint.min.l`
+  	opacity: 0.2;
+		box-shadow: 0px 0px 60px 20px rgb(0, 0, 0);
+		animation-name: shadow;
+		animation-duration: 3s;
+		animation-timing-function: linear;
+		animation-iteration-count: infinite;
+	`}
+
+	${breakpoint.min.l`
+    box-shadow: 0px 0px 15px 0px rgb(0, 0, 0);
```

Comment:
indentation :eyeglasses:

Reply:
I have not found any solution to lint & auto-format styled-component code on vscode...
Have to do it manually. Good catch.